app/assets/stylesheets/geoblacklight/modules/downloads.scss

```diff
@@ -47,3 +47,9 @@
     }
   }
 }
+
+.disabled {
+  cursor: not-allowed;
+  opacity: 0.5;
+  text-decoration: none;
+}
```

Comment:
could we make this `a.disabled` or something? seems like there's a chance it could conflict with other styles (bootstrap?) that have a more generic `disabled` class